library/src/amd_detail/hipblaslt-ext-op-internal.hpp

```diff
@@ -501,7 +501,15 @@ struct TensileLite::Serialization::MappingTraits<hipblaslt_ext::AMaxSolution, IO
             throw std::runtime_error("Invalid datatype in ext op library");
         }
 
-        if(scaleDatatypeStr == "F8")
+        if(scaleDatatypeStr == "F8N")
+        {
+            s.scaleDatatype = TensileLite::DataType::Float8_fnuz;
+        }
+        else if(scaleDatatypeStr == "B8N")
+        {
+            s.scaleDatatype = TensileLite::DataType::BFloat8_fnuz;
+        }
+		else if(scaleDatatypeStr == "F8")
```

Comment:
have you applied clang-format?

Reply:
Oops, I have not, I will apply, thanks!

Reply:
It looks like running clang format will introduce many changes in parts of code not touched in this PR and make reviewing the changes harder. 

For now, I've done basic formatting for files modified in this PR (removed trailing spaces, convert tabs -> spaces). I think it would be better to run clang format on all source files in a separate PR.